grammar.js

```diff
@@ -356,7 +374,7 @@ module.exports = grammar({
       optional(
         seq(
           'in',
-          commaSep1($._expression)
+          $.expression_list
```

Comment:
I think this change better represents Python's grammar, in that a list of comma separated expressions is what Python considers an expression list.